Homework_7_Java_Arrays_Loops/JavaArraysAndLoops/src/StudentSearch.java

```diff
@@ -0,0 +1,50 @@
+public class StudentSearch {
+    /*
+    Опис задачі:
+У вашому списку студентів ви хочете знайти конкретного студента за його номером залікової книжки.
+Напишіть програму, яка використовує цикл for each для перегляду списку студентів і знаходить номер залікової книжки,
+який ви шукаєте. Якщо студент знайдений, програма повинна вивести повідомлення і завершити роботу.
+Якщо студент не знайдений, програма повинна вивести відповідне повідомлення і завершити роботу.
+Вхідні дані:int[] studentIDs = {12345, 67890, 54321, 98765, 24680};
+
+int searchID = 54321;
+Очікуваний результат: у разі, якщо номер залікової книжки був знайдений у списку - вивести повідомлення про це і завершити роботу -
+якщо такої заліковки немає у списку - вивести відповідне повідомлення і завершити роботу
+     */
+
+    //for each
+
+    public static void main(String[] args) {
+        int[] studentIDs = {12345, 67890, 54321, 98765, 24680};
+        int searchID = 54321;
+        boolean isFound = false;
+
+        for (int i : studentIDs) {
+            if (i == searchID) {
+                System.out.println("This student is found");
+                isFound = true;
+                break;
+            }
+        }
+        if (!isFound) {
+            System.out.println("This student is not found");
+        }
+
+
+        //for
+
+
+        for (int j = 0; j < studentIDs.length; j++) {
```

Comment:
There's a small bug in the body of this loop

Hint: if the desired Id is having index 1, 3, 5, 7, ... -> the script will print "Not found" even if the ID is there inside the array

Reply:
![image](https://github.com/tetiana-qa7/QA_Automation_Java/assets/16562135/a4b8b640-a1fd-422a-b62e-1c5a797559be)